client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/EMController.java

```diff
@@ -65,7 +65,7 @@ public class EMController {
 
     private static final String htmlWarning =
             //ah! the beauty and elegance of Java when one just wants read a text resource as a string...
-            new Scanner(EMController.class.getResourceAsStream("/warning.html"), "UTF-8").useDelimiter("\\A").next();
+            new Scanner(Objects.requireNonNull(EMController.class.getResourceAsStream("/warning.html")), "UTF-8").useDelimiter("\\A").next();
```

Comment:
this one is a bit weird... Scanner would fail if input is null... and input can never be null (unless bug)